fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -805,7 +821,15 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 			writeThread = createGenericThreadPool();
 			readThreads = createGenericThreadPool();
 		}
-		writeThread->addThread(new Writer(db, id), "fdb-rocksdb-wr");
+
+		if (SERVER_KNOBS->ROCKSDB_PERFCONTEXT_ENABLE) {
+			perf = rocksdb_perfcontext_create();
```

Comment:
Is there any reason for creating this perf object here instead in rocksDBMetricLogger() ? Just to avoid passing this object across functions.

Reply:
> Is there any sample trace event on how this looks like in output?
It's like this in my local test:

user_key_comparison_count = 2523995, get_read_bytes = 800522, write_wal_time = 1091374671, get_snapshot_time = 164965027, get_from_memtable_time = 219041375, get_from_memtable_count = 49569, get_post_process_time = 167867328, write_pre_and_post_process_time = 344340566, write_memtable_time = 165398431, write_scheduling_flushes_compactions_time = 80630966